src/activities/hoc2019/SpritesheetFish.jsx

```diff
@@ -1,22 +1,43 @@
+/* eslint-disable react/prop-types */
```

Comment:
@madelynkasula I needed to disable `react/prop-types` linting on this file as we're importing the shape. I'm going to merge this but would like to get your thoughts on this!